Project-Aurora/Project-Aurora/Devices/Clevo/ClevoDevice.cs

```diff
@@ -36,9 +37,9 @@ class ClevoDevice : DefaultDevice
 
         public override string DeviceName => "Clevo Keyboard";
 
-        public override bool Initialize()
+        protected override Task<bool> DoInitialize()
```

Comment:
![image](https://user-images.githubusercontent.com/34775205/221385963-12d68b76-ebdf-4915-898d-9596d73fcc6a.png)
That should be a compiler warning CS1998.
I think almost all warnings have been supressed due to this setting in the .csproj:
```xml
    <WarningLevel>1</WarningLevel>
```
I think in the future, it might make sense to enable warns and explicitly `nowarn` on those we should ignore, and/or add TODOs to fix them one class at a time.

Reply:
I agree, didn't know this had that effect